src/coleslaw.lisp

```diff
@@ -3,16 +3,27 @@
 (defvar *last-revision* nil
   "The git revision prior to the last push. For use with GET-UPDATED-FILES.")
 
+(defvar *templating-engine* nil
```

Comment:
I would prefer the templating engine (and any related vars like `*djula-post-template*` and `*djula-index-template*`) be kept in the config object if at all possible. Sure, it's one big nasty God object, but it's in keeping with the conventions elsewhere in the project.